CMakeLists.txt

```diff
@@ -78,9 +78,31 @@ if (BUILD_SDF)
   # available during build time
   set(GZ_TOOLS_VER 2)
 
+  #################################################
+  # Find python
+  if (SKIP_PYBIND11)
+    message(STATUS "SKIP_PYBIND11 set - disabling python bindings")
+    find_package(Python3 COMPONENTS Interpreter)
+  else()
+    find_package(Python3 COMPONENTS Interpreter Development
+    )
+    if (NOT Python3_Development_FOUND)
+      GZ_BUILD_WARNING("Python development libraries are missing: Python interfaces are disabled.")
+    else()
+      set(PYBIND11_PYTHON_VERSION 3)
+      find_package(pybind11 2.4 CONFIG QUIET)
```

Comment:
Dobule checking: do we prefer to call it with `QUITE` and add custom STATUS messages below instead of removing the `QUIET` and the messages?

Reply:
it changes the formatting of the console messages, and I think it's subjective; I'm flexible on this

Reply:
up to you, no strong feeling either.

Reply:
since this is a backport; I'm going to leave this as is to match the behavior in newer branches